app/views/user_mailer/admin_hidden_work_notification.html.erb

```diff
@@ -1,12 +1,17 @@
 <% content_for :message do %>
-    <p>Dear <%= @user.default_pseud.byline %>,</p>
+  <p><%= t(".hello", name: style_bold(@user.login)).html_safe %></p>
 
-    <p>Your work <i><%= style_link(@work.title.html_safe, work_url(@work)) %></i> has been reported as being in violation of the Archive of Our Own's <%= style_link("Terms of Service", tos_url) %>.</p>
+  <p><%= t(".html.part1", title: style_creation_link(@work.title, @work)).html_safe %></p>
 
-    <p>We are investigating the matter and will contact you once the investigation has been completed. While the work is under investigation it has been hidden; only you will be able to access it.</p>
+  <p><%= t '.html.part2' %></p>
 
-    <p>If you have any questions, please <%= abuse_link("contact our Abuse Team") %>.</p>
+  <p><%= t '.html.part3' %></p>
 
-    <p>Sincerely,</p>
-    <p><%= style_bold("AO3 Abuse") %></p>
-<% end %>
+  <p><%= t('.html.part4', tos_link: tos_link(t '.tos')).html_safe %></p>
+
+  <p><%= t('.html.part5', abuse_link: abuse_link(t '.abuse_committee')).html_safe %></p>
```

Comment:
To help Translation out, could you update the keys for the links and URLs per [the guidelines](https://github.com/otwcode/otwarchive/wiki/Internationalization-(i18n)-Standards)?

>**If the variable represents a hyperlink, its name should end with _link.** This helps our translators identify links and the linked text. For example, `<p>If you have questions, please <%= link_to("contact Support", new_feedback_report_url) %>.</p>` can be replaced with `<p><%= t(".questions", support_link: link_to(t(".support", new_feedback_report_url)).html_safe) %></p>`. Note that the variable is the key for the linked text (`support`) plus the suffix` _link`.
>
>**If the variable represents a URL, its name should end with _url.** Particularly with mailers, this ensures links and URLs have distinct keys. For example, `If you have questions, please contact Support: <%= new_feedback_report_url %>.` can be replaced with `<%= t(".questions", support_url: new_feedback_report_url) %>.`

So here instead of `<%= t('.html.part5', abuse_link: abuse_link(t '.abuse_committee')).html_safe %>`, it would be: 

`<%= t(".html.part5", contact_abuse_link: abuse_link(t(".contact_abuse"))).html_safe %>`

(I changed the name because I thought it would be clearer if the variable didn't have the same name as the link helper method.)

And then in the text email, instead of `<%= t('.text.part5', abuse_link: 'http://archiveofourown.org/abuse_reports/new') %>`, it would be: 

`<%= t(".text.part5", contact_abuse_url: new_abuse_report_url) %>`